source/renderer/app/stores/HardwareWalletsStore.js

```diff
@@ -1738,6 +1739,13 @@ export default class HardwareWalletsStore extends Store {
     return type;
   };
 
+  _getTtl = () => {
+    const { networkTip } = this.stores.networkStatus;
+    const absoluteSlotNumber = get(networkTip, 'absoluteSlotNumber', 0);
```

Comment:
@tomothespian we need to handle the missing `absoluteSlotNumber` better...
Perhaps throw an error here?